src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1948,7 +1949,13 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
                         }
                         isVisibleResponse = false;
                     }
-
+                }
+                
+                if (question.questionType == FeedbackQuestionType.CONTRIB) {
```

Comment:
Ok you can add that extra table in then. Personally I think that these section panels shouldn't appear, but I guess this requires further discussion. Even if we change it, we should probably change this behaviour in a different issue anyway.